src/App.js

```diff
@@ -1,16 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+// import ColorChoice from './components/ColorChoice';
```

Comment:
Be sure to remove unused imports before opening a PR. In industry, when you open a PR your code should be in a state as if it were ready to be merged into production code. Therefore we want to get rid of unused comments and imports to keep the file from getting cluttered.